src/test/java/com/scalyr/integrations/kafka/AddEventsClientTest.java

```diff
@@ -420,14 +423,24 @@ public void testTooLargeAddEventsSkipped() throws Exception {
     AddEventsClient addEventsClient = new AddEventsClient(scalyrUrl, API_KEY_VALUE, ADD_EVENTS_TIMEOUT_MS, ADD_EVENTS_RETRY_DELAY_MS, compressor);
     List<Event> events = createTestEvents(numEvents, numServers, numLogFiles, numParsers);
     events.forEach(event -> event.setMessage(largeMsg));
-    addEventsClient.log(events);
+
+    assertEquals(0, server.getRequestCount());
+
+    addEventsResponse = addEventsClient.log(events).get(5, TimeUnit.SECONDS);
```

Comment:
Nit: better to declare variable where it is used.
`AddEventsResponse addEventsResponse = addEventsClient.log(events).get(5, TimeUnit.SECONDS);`